go/staking/api/api.go

```diff
@@ -415,6 +408,18 @@ type ConsensusParameters struct {
 	DisableTransfers       bool                         `json:"disable_transfers,omitempty"`
 	DisableDelegation      bool                         `json:"disable_delegation,omitempty"`
 	UndisableTransfersFrom map[signature.PublicKey]bool `json:"undisable_transfers_from,omitempty"`
+
+	// The proportion of fees disbursed to entities of the nodes that voted for a block.
+	FeeWeightVote int64 `json:"fee_weight_vote,omitempty"`
+	// The proportion of fees disbursed to the entity of the node that proposed a block.
+	FeeWeightPropose int64 `json:"fee_weight_propose,omitempty"`
```

Comment:
Add sanity checks that the sum of both weights is greater than zero (as otherwise the check in `disburseFees` will always fail).

Reply:
I gotta think about this

Reply:
yeah, that. and that FeeWeightVote should be nonnegative